README.md

```diff
@@ -18,6 +23,13 @@ Open a notepad in the Firefox toolbar to store notes about your browsing.
 | `npm run build` | Builds the application as a Web Extension.
 | `npm start`     | Launches Firefox with the Web Extension.
 
+NOTE: if you are running on Windows, and experience a build error use `$ npm build`
+
+Step 3:Load about:debugging#addons in the Firefox address bar.
+
+Step 4:Select a Firefox .xpi file or Manifest file under the src folder and click open afterwards
```

Comment:
Thank you for the review, yes it is not needed. For some reason I had to load it manually...removing those lines.Thanks again